src/main/java/de/dennisguse/opentracks/settings/PreferencesUtils.java

```diff
@@ -143,6 +143,15 @@ private static boolean getBoolean(int keyId, boolean defaultValue) {
         return sharedPreferences.getBoolean(getKey(keyId), defaultValue);
     }
 
+    public static String getWeight() {
+        return getString(R.string.settings_profile_Weight_key,"" );
```

Comment:
The requirement:
Weight (pounds or kg depending Settings->Default Units…): 

My feedback:
Should add label indicating the unit (lb or kg) in the dialog/text field. Use getUnitSystem() in PreferencesUtils to get the current setting.